src/main/java/com/google/sps/servlets/MailServlet.java

```diff
@@ -79,10 +82,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 * This function creates an email and sends it to the user
 **/
 
-  private String sendMultipartMail(String from, String to, String link) {
+  private String sendMultipartMail(String from, String to, String title, String message, String image) {
     Properties props = new Properties();
     Session session = Session.getDefaultInstance(props, null);
-
+    final String POSTCARD_CONTAINER = "<div class='pcard-container' id='pcard-design' " +
```

Comment:
Constants are static final, see naming guidelines at
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names
You can move that out into a static final field

Reply:
Done